API-doc-FORD-file.md

```diff
@@ -0,0 +1,51 @@
+---
+project: Fortran-lang/stdlib
+summary: A community driven standard library for (modern) Fortran
+src_dir: src
+exclude_dir: src/tests
```

Comment:
I excluded the tests from the documentation. We could include them if we wish but processing takes longer and it might dilute the API documentation with other stuff.

Reply:
I agree with not including the tests in the API documentation. Furthermore, specs include some examples for each procedure.